src/nginx.go

```diff
@@ -31,7 +36,23 @@ var (
 )
 
 func main() {
-	i, err := integration.New(integrationName, integrationVersion, integration.Args(&args))
+	var i *integration.Integration
+	var err error
+	cachePath := os.Getenv("NRIA_CACHE_PATH")
```

Comment:
I would move into a new function instead of having everything in the main func:

See this PR:
https://github.com/newrelic/nri-redis/pull/17/files#diff-118b55cb154646993f79f78a6b5efe3eR92